ImageGalery/GalleryCollectionViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  GalleryCollectionViewController.swift
+//  ImageGalery
+//
+//  Created by apple on 18/07/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "GalleryCollectionViewCell"
+
+
+protocol galleryUpdateDelegate {
+    func UppdateGalery(_ updatedGallery: ImageGallery)
+}
+
+class GalleryCollectionViewController: UICollectionViewController, UICollectionViewDropDelegate, UICollectionViewDelegateFlowLayout {
```

Comment:
Put the delegate methods in separate 'GalleryCollectionViewController' extension each